elektronLogging/pcpLogger.go

```diff
@@ -0,0 +1,50 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
+)
+
+type PcpLogger struct {
+	LoggerImpl
+}
+
+func NewPcpLogger(logType int, prefix string) *PcpLogger {
+	pLog := &PcpLogger{}
+	pLog.Type = logType
+	pLog.SetLogFile(prefix)
+	return pLog
```

Comment:
Why not just do this?
```go
return &PCPLogger{ 
    LoggerIml{
        Type: logType
        LogFile: CreateLogFile(prefix),
    }
}
```

Where `func CreateLogFile(prefix string) string` replaces `func SetLogFile(prefix string)`.